wmfdata/hive.py

```diff
@@ -63,77 +65,23 @@ def run_cli(
     # error ourselves. However, there's no simple way to determine if multiple
     # queries have been passed or separate their output, so it's not worth
     # the effort.
-    merged_commands = ";\n".join(commands)
+    commands = ";\n".join(ensure_list(commands))
+
+    connect_kwargs = {
+        'host': HIVE_URL,
```

Comment:
Please use double quotes, just to stay consistent 😊

Reply:
oh, I saw single quotes in a few places and wasn't sure.  Double quotes is fine, done.